scss/_header.scss

```diff
@@ -0,0 +1,151 @@
+@import 'variable';
+
+.header {
+  height: 150px;
+
+  .navbar-element {
+    background-color: #ffffff !important;
+    margin: 0;
+    padding: 0;
+    align-items: center;
+
+    .navbar-branch-element {
+      width: 340.16px;
```

Comment:
cái ảnh logo này cần cho width to như thế à?

Reply:
cái width này em cũng lấy thông số trong figma ạ